api/src/main/java/org/pmiops/workbench/api/OfflineRuntimeController.java

```diff
@@ -292,6 +298,38 @@ public ResponseEntity<Void> checkPersistentDisks() {
   private boolean notifyForUnusedDisk(LeonardoListPersistentDiskResponse disk, int daysUnused)
       throws MessagingException {
     String googleProject = leonardoMapper.toGoogleProject(disk.getCloudContext());
+    StringBuilder status = new StringBuilder();
+    try {
+      // If Label contains key aou-app-type it is either cromwell or Rstudio else its Jupyter
+      // runtime
+      if (((Map) disk.getLabels()).containsKey(LEONARDO_LABEL_APP_TYPE)) {
+        List<UserAppEnvironment> appEnviornments =
+            leonardoApiClient.listAppsInProjectAsService(googleProject);
+        if (appEnviornments != null && appEnviornments.size() > 0) {
+          // Some apps in list especially  in error state can have no disk
+          appEnviornments.stream()
```

Comment:
In your current email template, seems the logic is:
1: If disk is attached to app, email is `blabla and it is associated with the {app} environment `
2: If disk is not attached to app, email is `blabla and it is associated with the **Detached** environment `

2 might be a bit confusing. Two things we can do: 
2.1: change the wording to make it less confusing. 
2.2: conditionally show different paragrapth using `style="display:none;"` And we change text by detached or not 
2.3: Not show App status, just show Attached/Detached. that can make the text easier.